crates/server/src/admin/service.rs

```diff
@@ -125,6 +129,26 @@ pub(crate) fn setup(
         .route("/contexts", get(get_contexts::handler))
         .route("/identity/keys", delete(delete_auth_keys_handler))
         .route("/generate-jwt-token", post(generate_jwt_token_handler))
+        .route(
+            "/contexts/:context_id/proposals/:proposal_id/approvals/count",
+            get(get_number_of_proposal_approvals_handler),
+        )
+        .route(
+            "/contexts/:context_id/proposals/:proposal_id/approvals/users",
+            get(get_proposal_approvers_handler),
+        )
+        .route(
+            "/contexts/:context_id/proposals/count",
+            get(get_number_of_active_proposals_handler),
+        )
+        .route(
+            "/contexts/:context_id/proposals",
+            get(get_proposals_handler),
+        )
```

Comment:
why not just return the size in proposals? If you have all you just use `len` and if they come paginated it should come in the response